src/common/lru_cache.h

```diff
@@ -279,6 +302,24 @@ bool LRUCache<K, V, KeyTraits, ValueTraits>::Get(const K &key, V *value) {
     return true;
 }
 
+template <typename K,  typename V, typename KeyTraits, typename ValueTraits>
+bool LRUCache<K, V, KeyTraits, ValueTraits>::GetFirst(const V value, K *key) {
+    ::curve::common::WriteLockGuard guard(lock_);
+    if (ll_.empty()) {
+        return false;
+    }
+    auto it = ll_.rbegin();
```

Comment:
Since you use rbegin to traverse from the back, should this function be called GetLast

Reply:
fix

Reply:
fix